### PR TITLE
Ruby on Rails Captstone Project - Budged App (Spend Smart Sidekick)

### PR DESCRIPTION
**Pull request by ulises2607**
## Features Implemented
list of implemented features on project requirements

🎥 Presentation
Watch the presentation of the project [here](https://www.loom.com/share/904748623a264044bdddef260bc09d7c)

🕹 Demo
Test the demo of the project [here](https://spend-smart-sidekick.onrender.com/)

📝 ##General Requirements
✅ No linters errors.
✅ Used Gitflow methodology.
✅ Added a comprehensive and professional README file
📝 EEProject Requirements
✅ Defined main colors.
✅ Set custom styles and thanks the author
✅ Created model, migration, and test files for User, Expense, and Group tables with their relationship.
✅ Created Splash screen controller and view
✅ Configurated Devise to send email confirmation.
✅ Created Login and Sign Up views.
✅ Sign up by using your Full name, email, and password.
✅ Sign In by using email and password.
✅ Created Categories/Group page.
✅ Showed the name, icon, and total for each category.
✅ Click on a category item redirects the user to the transaction page.
✅ Added a button Add a new category in the categories index page.
✅ Created Add new category page and form with name and icon.
✅ Redirected to the home page when successfully added a category.
✅ Handled correctly the back button from the view.
✅ Created Payments (Expenses in the model to avoid errors) page.
✅ Added button Add a New Payment.
✅ Redirected users when clicking on the back button.
✅ Listed in most recent order.
✅ Created Add a New Payment page.
✅ Added button Save with name and amount.
✅ Added back buttons.
✅ Refactor code for ExpensesController to improve eager loading.
✅ Refactor code for CategoriesController to improve eager loading.
✅ Created Unit Tests for User, Categories, and Expenses models using rspec.
✅ Created Integration Tests for Splash, Categories, and Movements controllers using rspec and capybara.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,186 @@
...
+- **see the money spent on each category.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- https://spend-smart-sidekick.onrender.com/
+
+- https://www.loom.com/share/f4b6c8c1685b45539a134f9f743e7580
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] Amazing Job with the presentation 👏 
- [ ] However, The video  Does not contains a highlight about a piece of the code you found interesting or is proud of making.
- [ ] please add that to the video as the requirement states

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @ulises2607  ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits
* ✅  Strong Pull Request 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ulises2607) [reply README.md]: @mwanawabangona  Thank you!  Yes I forgot this because I couldn't preview the loom time to record 😅

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @ulises2607, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Requested changes implemented 🔥 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**